scripts/dashboard/labs.txt

```diff
@@ -14,27 +14,26 @@ c02-network08
 c02-network09
 c02-network10
 c02-network11
+c04-iam01
```

Comment:
If #1091 gets merged I think this will need to change too

Reply:
yeah.. but we decided to not change the name of that lab yet..maybe in the future..

Reply:
just replied to you there @roperto